yjit/src/invariants.rs

```diff
@@ -69,7 +62,7 @@ impl Invariants {
 pub fn assume_bop_not_redefined(jit: &mut JITState, ocb: &mut OutlinedCb, klass: RedefinitionFlag, bop: ruby_basic_operators) -> bool {
     if unsafe { BASIC_OP_UNREDEFINED_P(bop, klass) } {
         jit_ensure_block_entry_exit(jit, ocb);
-        Invariants::get_bop_assumptions(klass, bop).push(jit.get_block());
+        Invariants::get_instance().basic_operators.entry((klass, bop)).or_insert(Vec::new()).push(jit.get_block());
```

Comment:
Please don't have me but, I find that one-liners like this can be pretty hard to read. Can we make this code more readable?